roles/pulp_content_guard_rbac/README.md

```diff
@@ -0,0 +1,41 @@
+pulp_content_guard_rbac
+================
+
+This role creates and deletes Pulp RBAC content guards using the Pulp API
+
+Role variables
+--------------
+
+* `pulp_url`: URL of Pulp server. Default is `https://localhost:8080`
+* `pulp_admin_username`: Username used to access Pulp server. Default is `admin`
+* `pulp_admin_password`: Password used to access Pulp server. Default is unset
+* `pulp_content_guards_rbac_present`: List of groups to be present. Default is an empty list.
+* `pulp_content_guards_rbac_absent`: List of groups to be absent. Default is an empty list.
```

Comment:
It's more typical in Ansible (and this collection) to have a single list with a `state` attribute that defaults to present but can also be set to absent.

Reply:
The supported attributes of each item in the list should be documented (see other roles' READMEs)